DOCS:  add for CONCURRENCY_VERIFIED (#3510)
StackExchange/dnscontrol: documentation/language-reference/CONCURRENCY_VERIFIED.md

---
name: CONCURRENCY_VERIFIED
---

✅  - A checkmark means "this has been tested, and will run concurrently".  

> ❔  - The questionmark means "it hasn't been tested, so we don't know"  

Reviewer: ```suggestion
❔  - The questionmark means "it hasn't been tested, safety unknown"  
```
